SWLOR.Game.Server/Feature/PlaceableScripts.cs

```diff
@@ -157,5 +157,26 @@ public static void PurchaseRebuild()
 
             SendMessageToPC(player, $"Remaining rebuild tokens: {Currency.GetCurrency(player, CurrencyType.RebuildToken)}");
         }
+
+        /// <summary>
+        /// When the module loads, sets the Cursed flag onto all items within a placeable with the local string 'viewonly_storage'.
+        /// </summary>
+        [NWNEventHandler("mod_load")]
```

Comment:
Why is this scripted out? You can accomplish the same thing by placing the placeable down, adding the book to the inventory, then marking that instance of the item as undroppable. Should be no reason to iterate through every single object in the module to accomplish this and will increase boot times.